RadixWallet/Features/TransactionReviewFeature/TransactionReviewAccount/TransactionReviewAccount.swift

```diff
@@ -23,7 +23,7 @@ public struct TransactionReviewAccounts: Sendable, FeatureReducer {
 
 	public enum DelegateAction: Sendable, Equatable {
 		case showCustomizeGuarantees
-		case showAsset(TransactionReview.Transfer)
+		case showAsset(TransactionReview.Transfer, OnLedgerEntity.NonFungibleToken?)
```

Comment:
A bit unfortunate, but that is the currently the most proper way to transfer up the selected token.